### ISSUE TITLE
Make --previous-metadata optional

### ISSUE THREAD

**Issue by xeruf** [novoid/lazyblorg#75]
While testing out, I don't have separate staging and build environments, so that flag does not help me much.
The script can deal with the file not existing, why not allow to do without the flag at all?

**Comment by novoid** [comment]
Good point. For taking a closer look, I have to re-learn about this mechanism of mine.

As you've noticed already, you're dealing with a "works for me"-project that comes with certain drawbacks. ;-)
